recipes/kedro-airflow/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "kedro-airflow" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/kedro-airflow-{{ version }}.tar.gz
```

Comment:
The tarball is missing a file that is used in the setup.py. See https://github.com/kedro-org/kedro-plugins/blob/main/kedro-airflow/setup.py#L38

You can:
1. patch it here
2. fix upstream and wait for a new release
3. try the github source instead of the PyPI one